### PR TITLE
Changes for classification 

### PR DESCRIPTION
**Pull request by swetha097**

### CODE REVIEW

**Comment by LakshmiKumar23** [comment]
@swetha097 can you check the lgtm errros?

File: rocAL/rocAL_pybind/amd/rocal/decoders.py [hunk 1]
```diff
@@ -177,10 +179,9 @@ def image_random_crop(*inputs,user_feature_key_map=None ,path = '', file_root= '
             'is_output': False,
             "shuffle": random_shuffle,
             "loop": False,
-            "decode_size_policy": types.MAX_SIZE,
-            "max_width": 0,
-            "max_height":0,
-            "dec_type": types.DECODER_TJPEG}
+            "decode_size_policy": types.USER_GIVEN_SIZE,
+            "max_width": 2000,
+            "max_height":2000}
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/amd/rocal/decoders.py:184]: @swetha097 why are we setting the max_width and max_height in this case?

File: rocAL/rocAL_pybind/example/new_api/rocAL_api_python_unittest.py [hunk 2]
```diff
@@ -59,7 +59,15 @@ def main():
 
 
         if augmentation_name == "resize":
-            output = fn.resize(images, resize_x=300, resize_y=300)
+            images = fn.resize_shorterimages, resize_size=256)
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/example/new_api/rocAL_api_python_unittest.py:62]: @swetha097 seems like a '(' is missing

Comment (swetha097) [reply rocAL/rocAL_pybind/example/new_api/rocAL_api_python_unittest.py]: Resolved in latest commit

**Review by LakshmiKumar23 — APPROVED** [review]
@kiritigowda  @swetha097 image classification training passes on new docker. 
Can be merged if CI passes

**Comment by LakshmiKumar23** [comment]
@swetha097 Seems like this PR builds only with RPP master. Not with the released tag - 0.93
Also, please sync with TOT

**Comment by kiritigowda** [comment]
@swetha097 can you fix the merge conflicts

**Comment by kiritigowda** [comment]
@LakshmiKumar23 let me know when this is ready.

**Comment by LakshmiKumar23** [comment]
@kiritigowda Ran a training. It passes. Can be merged

**Review by LakshmiKumar23 — APPROVED** [review]

**Comment by LakshmiKumar23** [comment]
> @kiritigowda Ran a training. It passes. Can be merged

@kiritigowda Can we merge this PR?

**Comment by kiritigowda** [comment]
@LakshmiKumar23 trying to run the codacy. Will merge once that is resolved.

**Comment by kiritigowda** [comment]
@swetha097 @LakshmiKumar23 -- reopen this PR with fixes to TOT build.

**Pull request merged**